build.js

```diff
@@ -31,6 +31,10 @@ var child_process = require('child_process');
 
 var useCompilers = String(process.argv[2]).toLowerCase() === "compilers";
 
+var isOptional = function isOptional(category) {
```

Comment:
I just always prefer function declarations unless in blocks. Not a big deal, was just curious. LGTM, merging.